src/main.cpp

```diff
@@ -59,6 +59,11 @@ static void SetupCallbacks(ListView& albumViewRef, ListView& songViewRef)
             songVec.push_back({ cols, file.path() });
         }
 
+        // Sort the items alphabetically
+        std::sort(songVec.begin(), songVec.end(), [](const auto& a, const auto& b) {
```

Comment:
It would be nice if you made std::sort into std::ranges::sort, i try to be consistent with that.